README.md

```diff
@@ -14,26 +14,99 @@ npm install throttle-debounce --save
 
 ## Usage
 
+### `throttle`
+
 ```js
-import { throttle, debounce } from 'throttle-debounce';
+import { throttle } from 'throttle-debounce';
+
+const throttleFunc = throttle(300, () => {
+	// Throttled function
+});
 
-throttle(300, () => {
+// which is equals to, because noTrailing is default to false
+const throttleFunc = throttle(1000, false, () => {
 	// Throttled function
 });
 
-debounce(300, () => {
+// example
+
+const throttleFunc = throttle(1000, false, (num) => {
+  console.log('num:', num)
+});
+
+throttleFunc(1) // will execute the callback
+throttleFunc(2) // won't execute callback
+throttleFunc(3) // won't execute callback
+
+// will execute the callback, because noTrailing is false, 
+// but if we set noTrailing as true, this callback won't be executed.
+throttleFunc(4)
+
+setTimeout(() => {
+  throttleFunc(10) // will execute the callback
+}, 1200)
+
+// output
+// num: 1
+// num: 4
+// num: 10
+```
+
+### `debounce`
+
+```js
+import { debounce } from 'throttle-debounce';
+
+const debounceFunc = debounce(300, () => {
+	// Debounced function
+});
```

Comment:
@itboos this is not needed since following code has actual example

Reply:
ok, I will delete this later and separate ES6 changes in a new  PR.